ezgui/src/backend_glow.rs

```diff
@@ -312,19 +289,32 @@ impl PrerenderInnards {
         }
     }
 
-    pub fn window_resized(&self, width: f64, height: f64) {
-        self.windowed_context
-            .resize(winit::dpi::PhysicalSize::new(width as u32, height as u32));
+    pub fn window_resized(&self, new_size: ScreenDims, scale_factor: f64) {
```

Comment:
I'm trying to prefer `ScreenFoo` structs rather than raw floats in more places as a sort of hint that the thing is in units of logical pixels.

Reply:
SGTM! Lots of the places dealing in raw (f64, f64)s, like canvas window_width and window_height, are just legacy